### PR TITLE
Use meaningful pose and give option of empty_cache

### PR DESCRIPTION
**Pull request by JisuHann**
# What is new?

1. Modify `sample_added_noise` to use `guess_translation` from FoundationPose original code. Rotation code is the same.
2. We give option for `empty_cache`.

### Position noise is also added. 
Case of not giving noise showed failure (only one case happened).
Failure case
tmp

When small amount of position noise is given, it success
tmp

### CODE REVIEW

File: estimater.py [hunk 1]
```diff
@@ -370,33 +377,40 @@ def track_one_among_noises(self, rgb, depth, K, iteration, pose_last, sample_num
...
+      pos = copy.deepcopy(org_pos).repeat(sample_n, 1)
+      ori = copy.deepcopy(org_ori).repeat(sample_n, 1)
+      pos_noise = torch.rand(size=(sample_n, 3)) * 2 -1 # sample from [-1, 1]
+      rot_noise = torch.rand(size=(sample_n, 4)) * 2 -1 # sample from [-1, 1]
+      # # ====================== Add noise ========================
+      if sample_guess_translation: # guess translation
+        pos = torch.tensor(self.guess_translation(depth=depth, mask=mask, K=K)).repeat(sample_n, 1)
+      else: # add noise on last timestep pose
+        position_noise = pos_noise * current_pos_noise
+        pos += position_noise
```
Comment (yycho0108) [inline estimater.py]: So the translation is always fixed and only rotation is sampled with noise? Any rationale for this?

**Review by yycho0108 — APPROVED** [review]
LGTM I guess

File: estimater.py (hunk 1 above)
Comment (JisuHann) [reply estimater.py]: We modified the code to give noise at translation (See updated comment above). So now noise is added at both translation and rotation.

**Comment by JisuHann** [comment]
Actually I do not have the permission for merge PR so @yycho0108 can you do this?

**Pull request merged**